### ISSUE TITLE
Fresh Install

### ISSUE THREAD

**Issue by EvilMonkey0** [brycelarge/xteve-vpn#5]
Hello, I downloaded and installed the container. I followed the setup and used PIA. The container launches but when I run the curl ifconfig.io command my actual IP shows instead of the one from the  pa server. I don't think I am connected through PIA.

Any support is appreciated. Been waiting along time for something like this.

Update
If I change to custom instead of PIA, remove the UDP and server it will grab a PIA IP address. I am unable to access the webui tho. :(

**Comment by brycelarge** [comment]
Hello,

Check the container logs and send me the output so I can see what’s going on?

Some things to check, make sure the container is in bridge mode otherwise the vpn will not start. 

Kind Regards 
Bryce

**Comment by EvilMonkey0** [comment]
> Hello, Check the container logs and send me the output so I can see what’s going on? Some things to check, make sure the container is in bridge mode otherwise the vpn will not start. Kind Regards Bryce
> […](#)
> On Nov 30, 2021, at 6:41 AM, EvilMonkey0 ***@***.***> wrote: ﻿ Hello, I downloaded and installed the container. I followed the setup and used PIA. The container launches but when I run the curl ifconfig.io command my actual IP shows instead of the one from the pa server. I don't think I am connected through PIA. Any support is appreciated. Been waiting along time for something like this. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing...
[cont-init.d] 30-config: exited 0.
[cont-init.d] 40-vpn-config: executing...
2021-11-30 05:05:42 [OpenVPN] config file ca_toronto.ovpn found
2021-11-30 05:05:42 [info] Adding 8.8.8.8 to /etc/resolv.conf
2021-11-30 05:05:42 [info] Adding 8.8.4.4 to /etc/resolv.conf
2021-11-30 05:05:42 [info] Adding route to local network 192.168.2.5 via 172.17.0.1 dev eth0
[cont-init.d] 40-vpn-config: exited 0.
[cont-init.d] 90-custom-folders: executing...
[cont-init.d] 90-custom-folders: exited 0.
[cont-init.d] 99-custom-scripts: executing...
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2021-11-30 05:05:42 [xTeVe] Waiting 10s for OpenVPN to boot before starting xTeVe
2021-11-30 05:05:42 [OpenVPN] config file ca_toronto.ovpn found
2021-11-30 05:05:42 [OpenVPN] using provider: CUSTOM
2021-11-30 05:05:42 [OpenVPN] Start Script: /usr/sbin/openvpn --reneg-sec 0 --disable-occ --auth-nocache --mute-replay-warnings --pull-filter ignore 'ping' --pull-filter ignore 'ping-restart' --pull-filter ignore 'route-ipv6' --pull-filter ignore 'ifconfig-ipv6 --pull-filter ignore 'tun-ipv6' --pull-filter ignore 'dhcp-option DNS6' --pull-filter ignore 'reneg-sec' --pull-filter ignore 'persist-tun' --pull-filter ignore 'block-outside-dns' --config /config/openvpn/ca_toronto.ovpn
Tue Nov 30 05:05:43 2021 OpenVPN 2.4.4 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Apr 27 2021
Tue Nov 30 05:05:43 2021 library versions: OpenSSL 1.1.1 11 Sep 2018, LZO 2.08
Tue Nov 30 05:05:43 2021 TCP/UDP: Preserving recently used remote address: [AF_INET]154.3.42.21:1198
Tue Nov 30 05:05:43 2021 UDP link local: (not bound)
Tue Nov 30 05:05:43 2021 UDP link remote: [AF_INET]154.3.42.21:1198
Tue Nov 30 05:05:43 2021 [toronto422] Peer Connection Initiated with [AF_INET]154.3.42.21:1198
Tue Nov 30 05:05:44 2021 TUN/TAP device tun0 opened
Tue Nov 30 05:05:44 2021 do_ifconfig, tt->did_ifconfig_ipv6_setup=0
Tue Nov 30 05:05:44 2021 /sbin/ip link set dev tun0 up mtu 1500
Tue Nov 30 05:05:44 2021 /sbin/ip addr add dev tun0 10.18.112.223/24 broadcast 10.18.112.255
Tue Nov 30 05:05:44 2021 Initialization Sequence Completed
2021-11-30 05:05:53 [xTeVe] Public IP: 154.3.42.21
2021-11-30 05:05:53 [xTeVe] Start Script: xteve -port=34400 -config=/config -branch=master -debug=0
Git Branch is now: master
For help run: ./xteve -h

2021/11/30 05:05:53 [xTeVe] Version: 2.2.0 Build: 0200
2021/11/30 05:05:53 [xTeVe] Database Version: 2.1.0
2021/11/30 05:05:53 [xTeVe] System IP Addresses: IPv4: 3 | IPv6: 0
2021/11/30 05:05:53 [xTeVe] Hostname: 3f77cf40f6b0
2021/11/30 05:05:53 [xTeVe] System Folder: /config/
2021/11/30 05:05:53 [xTeVe] Load Settings: /config/settings.json
2021/11/30 05:05:53 [xTeVe] Git Branch: Switching Git Branch to -> master
2021/11/30 05:05:53 [xTeVe] Temporary Folder: /tmp/xteve/2021-11-C4ER-WCA1T4/
2021/11/30 05:05:53 [xTeVe] GitHub: https://github.com/xteve-project
2021/11/30 05:05:53 [xTeVe] Git Branch: master [xteve-project]
2021/11/30 05:05:53 [xTeVe] SSDP / DLNA: true
2021/11/30 05:05:54 [xTeVe] UUID: 2021-11-C4ER-WCA1T4
2021/11/30 05:05:54 [xTeVe] Tuner (Plex / Emby): 3
2021/11/30 05:05:54 [xTeVe] EPG Source: XEPG
2021/11/30 05:05:54 [xTeVe] Plex Channel Limit: 480
2021/11/30 05:05:54 [xTeVe] Unfiltered Chan. Limit: 480
2021/11/30 05:05:54 [xTeVe] Backup Path: /config/backup/
2021/11/30 05:05:54 [xTeVe] Backup file: /config/backup/xteve_auto_backup_20211130_0505.zip
2021/11/30 05:05:54 [xTeVe] Download: ****redacted****
2021/11/30 05:05:54 [xTeVe] Header filename: tv_channels_9RWKyUJmY2_plus.m3u
2021/11/30 05:05:56 [xTeVe] Check File:****redacted****
2021/11/30 05:05:56 [xTeVe] Save File: ****redacted****
2021/11/30 05:05:56 [xTeVe] Download:****redacted****
2021/11/30 05:06:13 [xTeVe] Check File: ****redacted****
2021/11/30 05:06:15 [xTeVe] Save File:****redacted****
2021/11/30 05:06:15 [xTeVe] All streams: 7616
2021/11/30 05:06:15 [xTeVe] Active streams: 261
2021/11/30 05:06:15 [xTeVe] Filter: 1
2021/11/30 05:06:15 [xTeVe] XEPG: Parse XMLTV file: XMLTV
2021/11/30 05:06:16 [xTeVe] XEPG: Update database
2021/11/30 05:06:16 [xTeVe] XEPG: Save DB file
2021/11/30 05:06:16 [xTeVe] XEPG: Map channels
2021/11/30 05:06:16 [xTeVe] XEPG: Cleanup database
2021/11/30 05:06:16 [xTeVe] XEPG Channels: 232
2021/11/30 05:06:16 [xTeVe] DVR IP: 172.17.0.3:34400
2021/11/30 05:06:16 [xTeVe] Web Interface: http://172.17.0.3:34400/web/ | xTeVe is also available via the other 2 IP's.
2021/11/30 05:06:16 [xTeVe] XEPG: Create XMLTV file (/config/data/xteve.xml)
2021/11/30 05:06:17 [xTeVe] XEPG: Compress XMLTV file (/config/data/xteve.xml.gz)
2021/11/30 05:06:17 [xTeVe] XEPG: Create M3U file (/config/data/xteve.m3u)
2021/11/30 05:06:17 [xTeVe] XEPG: Ready to use

---------------------------------


If I remove the vpn file and leave it at custom, I can access the webui.

**Comment by brycelarge** [comment]
Hello,

 

I see OpenVPN connected successfully with IP 154.3.42.21:

Tue Nov 30 05:05:43 2021 [toronto422] Peer Connection Initiated with [AF_INET]154.3.42.21:1198


Then I see xTeVe connected with the same public IP:

2021-11-30 05:05:53 [xTeVe] Public IP: 154.3.42.21

 

I did a whois search on IP 154.3.42.21 and it is a Canada Toronto IP address.

 

To check the container is running through the VPN you can run the following from your host:

docker exec -t xteve-vpn sh -c 'curl ifconfig.me'

 

 

Kind Regards

Bryce

 

 

From: EvilMonkey0 ***@***.***>
Reply to: brycelarge/xteve-vpn ***@***.***>
Date: Tuesday, 30 November 2021 at 07:11
To: brycelarge/xteve-vpn ***@***.***>
Cc: ***@***.***" ***@***.***>, Comment ***@***.***>
Subject: Re: [brycelarge/xteve-vpn] Fresh Install (Issue #5)

 

Hello, Check the container logs and send me the output so I can see what’s going on? Some things to check, make sure the container is in bridge mode otherwise the vpn will not start. Kind Regards Bryce
…
On Nov 30, 2021, at 6:41 AM, EvilMonkey0 @.***> wrote: ﻿ Hello, I downloaded and installed the container. I followed the setup and used PIA. The container launches but when I run the curl ifconfig.io command my actual IP shows instead of the one from the pa server. I don't think I am connected through PIA. Any support is appreciated. Been waiting along time for something like this. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing...
[cont-init.d] 30-config: exited 0.
[cont-init.d] 40-vpn-config: executing...
2021-11-30 05:05:42 [OpenVPN] config file ca_toronto.ovpn found
2021-11-30 05:05:42 [info] Adding 8.8.8.8 to /etc/resolv.conf
2021-11-30 05:05:42 [info] Adding 8.8.4.4 to /etc/resolv.conf
2021-11-30 05:05:42 [info] Adding route to local network 192.168.2.5 via 172.17.0.1 dev eth0
[cont-init.d] 40-vpn-config: exited 0.
[cont-init.d] 90-custom-folders: executing...
[cont-init.d] 90-custom-folders: exited 0.
[cont-init.d] 99-custom-scripts: executing...
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2021-11-30 05:05:42 [xTeVe] Waiting 10s for OpenVPN to boot before starting xTeVe
2021-11-30 05:05:42 [OpenVPN] config file ca_toronto.ovpn found
2021-11-30 05:05:42 [OpenVPN] using provider: CUSTOM
2021-11-30 05:05:42 [OpenVPN] Start Script: /usr/sbin/openvpn --reneg-sec 0 --disable-occ --auth-nocache --mute-replay-warnings --pull-filter ignore 'ping' --pull-filter ignore 'ping-restart' --pull-filter ignore 'route-ipv6' --pull-filter ignore 'ifconfig-ipv6 --pull-filter ignore 'tun-ipv6' --pull-filter ignore 'dhcp-option DNS6' --pull-filter ignore 'reneg-sec' --pull-filter ignore 'persist-tun' --pull-filter ignore 'block-outside-dns' --config /config/openvpn/ca_toronto.ovpn
Tue Nov 30 05:05:43 2021 OpenVPN 2.4.4 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Apr 27 2021
Tue Nov 30 05:05:43 2021 library versions: OpenSSL 1.1.1 11 Sep 2018, LZO 2.08
Tue Nov 30 05:05:43 2021 TCP/UDP: Preserving recently used remote address: [AF_INET]154.3.42.21:1198
Tue Nov 30 05:05:43 2021 UDP link local: (not bound)
Tue Nov 30 05:05:43 2021 UDP link remote: [AF_INET]154.3.42.21:1198
Tue Nov 30 05:05:43 2021 [toronto422] Peer Connection Initiated with [AF_INET]154.3.42.21:1198
Tue Nov 30 05:05:44 2021 TUN/TAP device tun0 opened
Tue Nov 30 05:05:44 2021 do_ifconfig, tt->did_ifconfig_ipv6_setup=0
Tue Nov 30 05:05:44 2021 /sbin/ip link set dev tun0 up mtu 1500
Tue Nov 30 05:05:44 2021 /sbin/ip addr add dev tun0 10.18.112.223/24 broadcast 10.18.112.255
Tue Nov 30 05:05:44 2021 Initialization Sequence Completed
2021-11-30 05:05:53 [xTeVe] Public IP: 154.3.42.21
2021-11-30 05:05:53 [xTeVe] Start Script: xteve -port=34400 -config=/config -branch=master -debug=0
Git Branch is now: master
For help run: ./xteve -h

2021/11/30 05:05:53 [xTeVe] Version: 2.2.0 Build: 0200
2021/11/30 05:05:53 [xTeVe] Database Version: 2.1.0
2021/11/30 05:05:53 [xTeVe] System IP Addresses: IPv4: 3 | IPv6: 0
2021/11/30 05:05:53 [xTeVe] Hostname: 3f77cf40f6b0
2021/11/30 05:05:53 [xTeVe] System Folder: /config/
2021/11/30 05:05:53 [xTeVe] Load Settings: /config/settings.json
2021/11/30 05:05:53 [xTeVe] Git Branch: Switching Git Branch to -> master
2021/11/30 05:05:53 [xTeVe] Temporary Folder: /tmp/xteve/2021-11-C4ER-WCA1T4/
2021/11/30 05:05:53 [xTeVe] GitHub: https://github.com/xteve-project
2021/11/30 05:05:53 [xTeVe] Git Branch: master [xteve-project]
2021/11/30 05:05:53 [xTeVe] SSDP / DLNA: true
2021/11/30 05:05:54 [xTeVe] UUID: 2021-11-C4ER-WCA1T4
2021/11/30 05:05:54 [xTeVe] Tuner (Plex / Emby): 3
2021/11/30 05:05:54 [xTeVe] EPG Source: XEPG
2021/11/30 05:05:54 [xTeVe] Plex Channel Limit: 480
2021/11/30 05:05:54 [xTeVe] Unfiltered Chan. Limit: 480
2021/11/30 05:05:54 [xTeVe] Backup Path: /config/backup/
2021/11/30 05:05:54 [xTeVe] Backup file: /config/backup/xteve_auto_backup_20211130_0505.zip
2021/11/30 05:05:54 [xTeVe] Download: redacted
2021/11/30 05:05:54 [xTeVe] Header filename: tv_channels_9RWKyUJmY2_plus.m3u
2021/11/30 05:05:56 [xTeVe] Check File:redacted
2021/11/30 05:05:56 [xTeVe] Save File: redacted
2021/11/30 05:05:56 [xTeVe] Download:redacted
2021/11/30 05:06:13 [xTeVe] Check File: redacted
2021/11/30 05:06:15 [xTeVe] Save File:redacted
2021/11/30 05:06:15 [xTeVe] All streams: 7616
2021/11/30 05:06:15 [xTeVe] Active streams: 261
2021/11/30 05:06:15 [xTeVe] Filter: 1
2021/11/30 05:06:15 [xTeVe] XEPG: Parse XMLTV file: XMLTV
2021/11/30 05:06:16 [xTeVe] XEPG: Update database
2021/11/30 05:06:16 [xTeVe] XEPG: Save DB file
2021/11/30 05:06:16 [xTeVe] XEPG: Map channels
2021/11/30 05:06:16 [xTeVe] XEPG: Cleanup database
2021/11/30 05:06:16 [xTeVe] XEPG Channels: 232
2021/11/30 05:06:16 [xTeVe] DVR IP: 172.17.0.3:34400
2021/11/30 05:06:16 [xTeVe] Web Interface: http://172.17.0.3:34400/web/ | xTeVe is also available via the other 2 IP's.
2021/11/30 05:06:16 [xTeVe] XEPG: Create XMLTV file (/config/data/xteve.xml)
2021/11/30 05:06:17 [xTeVe] XEPG: Compress XMLTV file (/config/data/xteve.xml.gz)
2021/11/30 05:06:17 [xTeVe] XEPG: Create M3U file (/config/data/xteve.m3u)
2021/11/30 05:06:17 [xTeVe] XEPG: Ready to use

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by EvilMonkey0** [comment]
> Hello, I see OpenVPN connected successfully with IP 154.3.42.21: Tue Nov 30 05:05:43 2021 [toronto422] Peer Connection Initiated with [AF_INET]154.3.42.21:1198 Then I see xTeVe connected with the same public IP: 2021-11-30 05:05:53 [xTeVe] Public IP: 154.3.42.21 I did a whois search on IP 154.3.42.21 and it is a Canada Toronto IP address. To check the container is running through the VPN you can run the following from your host: docker exec -t xteve-vpn sh -c 'curl ifconfig.me' Kind Regards Bryce From: EvilMonkey0 ***@***.***> Reply to: brycelarge/xteve-vpn ***@***.***> Date: Tuesday, 30 November 2021 at 07:11 To: brycelarge/xteve-vpn ***@***.***> Cc: ***@***.***" ***@***.***>, Comment ***@***.***> Subject: Re: [brycelarge/xteve-vpn] Fresh Install (Issue #5) Hello, Check the container logs and send me the output so I can see what’s going on? Some things to check, make sure the container is in bridge mode otherwise the vpn will not start. Kind Regards Bryce … On Nov 30, 2021, at 6:41 AM, EvilMonkey0 @.***> wrote: ﻿ Hello, I downloaded and installed the container. I followed the setup and used PIA. The container launches but when I run the curl ifconfig.io command my actual IP shows instead of the one from the pa server. I don't think I am connected through PIA. Any support is appreciated. Been waiting along time for something like this. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. [cont-init.d] 10-adduser: exited 0. [cont-init.d] 30-config: executing... [cont-init.d] 30-config: exited 0. [cont-init.d] 40-vpn-config: executing... 2021-11-30 05:05:42 [OpenVPN] config file ca_toronto.ovpn found 2021-11-30 05:05:42 [info] Adding 8.8.8.8 to /etc/resolv.conf 2021-11-30 05:05:42 [info] Adding 8.8.4.4 to /etc/resolv.conf 2021-11-30 05:05:42 [info] Adding route to local network 192.168.2.5 via 172.17.0.1 dev eth0 [cont-init.d] 40-vpn-config: exited 0. [cont-init.d] 90-custom-folders: executing... [cont-init.d] 90-custom-folders: exited 0. [cont-init.d] 99-custom-scripts: executing... [custom-init] no custom files found exiting... [cont-init.d] 99-custom-scripts: exited 0. [cont-init.d] done. [services.d] starting services [services.d] done. 2021-11-30 05:05:42 [xTeVe] Waiting 10s for OpenVPN to boot before starting xTeVe 2021-11-30 05:05:42 [OpenVPN] config file ca_toronto.ovpn found 2021-11-30 05:05:42 [OpenVPN] using provider: CUSTOM 2021-11-30 05:05:42 [OpenVPN] Start Script: /usr/sbin/openvpn --reneg-sec 0 --disable-occ --auth-nocache --mute-replay-warnings --pull-filter ignore 'ping' --pull-filter ignore 'ping-restart' --pull-filter ignore 'route-ipv6' --pull-filter ignore 'ifconfig-ipv6 --pull-filter ignore 'tun-ipv6' --pull-filter ignore 'dhcp-option DNS6' --pull-filter ignore 'reneg-sec' --pull-filter ignore 'persist-tun' --pull-filter ignore 'block-outside-dns' --config /config/openvpn/ca_toronto.ovpn Tue Nov 30 05:05:43 2021 OpenVPN 2.4.4 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Apr 27 2021 Tue Nov 30 05:05:43 2021 library versions: OpenSSL 1.1.1 11 Sep 2018, LZO 2.08 Tue Nov 30 05:05:43 2021 TCP/UDP: Preserving recently used remote address: [AF_INET]154.3.42.21:1198 Tue Nov 30 05:05:43 2021 UDP link local: (not bound) Tue Nov 30 05:05:43 2021 UDP link remote: [AF_INET]154.3.42.21:1198 Tue Nov 30 05:05:43 2021 [toronto422] Peer Connection Initiated with [AF_INET]154.3.42.21:1198 Tue Nov 30 05:05:44 2021 TUN/TAP device tun0 opened Tue Nov 30 05:05:44 2021 do_ifconfig, tt->did_ifconfig_ipv6_setup=0 Tue Nov 30 05:05:44 2021 /sbin/ip link set dev tun0 up mtu 1500 Tue Nov 30 05:05:44 2021 /sbin/ip addr add dev tun0 10.18.112.223/24 broadcast 10.18.112.255 Tue Nov 30 05:05:44 2021 Initialization Sequence Completed 2021-11-30 05:05:53 [xTeVe] Public IP: 154.3.42.21 2021-11-30 05:05:53 [xTeVe] Start Script: xteve -port=34400 -config=/config -branch=master -debug=0 Git Branch is now: master For help run: ./xteve -h 2021/11/30 05:05:53 [xTeVe] Version: 2.2.0 Build: 0200 2021/11/30 05:05:53 [xTeVe] Database Version: 2.1.0 2021/11/30 05:05:53 [xTeVe] System IP Addresses: IPv4: 3 | IPv6: 0 2021/11/30 05:05:53 [xTeVe] Hostname: 3f77cf40f6b0 2021/11/30 05:05:53 [xTeVe] System Folder: /config/ 2021/11/30 05:05:53 [xTeVe] Load Settings: /config/settings.json 2021/11/30 05:05:53 [xTeVe] Git Branch: Switching Git Branch to -> master 2021/11/30 05:05:53 [xTeVe] Temporary Folder: /tmp/xteve/2021-11-C4ER-WCA1T4/ 2021/11/30 05:05:53 [xTeVe] GitHub: https://github.com/xteve-project 2021/11/30 05:05:53 [xTeVe] Git Branch: master [xteve-project] 2021/11/30 05:05:53 [xTeVe] SSDP / DLNA: true 2021/11/30 05:05:54 [xTeVe] UUID: 2021-11-C4ER-WCA1T4 2021/11/30 05:05:54 [xTeVe] Tuner (Plex / Emby): 3 2021/11/30 05:05:54 [xTeVe] EPG Source: XEPG 2021/11/30 05:05:54 [xTeVe] Plex Channel Limit: 480 2021/11/30 05:05:54 [xTeVe] Unfiltered Chan. Limit: 480 2021/11/30 05:05:54 [xTeVe] Backup Path: /config/backup/ 2021/11/30 05:05:54 [xTeVe] Backup file: /config/backup/xteve_auto_backup_20211130_0505.zip 2021/11/30 05:05:54 [xTeVe] Download: redacted 2021/11/30 05:05:54 [xTeVe] Header filename: tv_channels_9RWKyUJmY2_plus.m3u 2021/11/30 05:05:56 [xTeVe] Check File:redacted 2021/11/30 05:05:56 [xTeVe] Save File: redacted 2021/11/30 05:05:56 [xTeVe] Download:redacted 2021/11/30 05:06:13 [xTeVe] Check File: redacted 2021/11/30 05:06:15 [xTeVe] Save File:redacted 2021/11/30 05:06:15 [xTeVe] All streams: 7616 2021/11/30 05:06:15 [xTeVe] Active streams: 261 2021/11/30 05:06:15 [xTeVe] Filter: 1 2021/11/30 05:06:15 [xTeVe] XEPG: Parse XMLTV file: XMLTV 2021/11/30 05:06:16 [xTeVe] XEPG: Update database 2021/11/30 05:06:16 [xTeVe] XEPG: Save DB file 2021/11/30 05:06:16 [xTeVe] XEPG: Map channels 2021/11/30 05:06:16 [xTeVe] XEPG: Cleanup database 2021/11/30 05:06:16 [xTeVe] XEPG Channels: 232 2021/11/30 05:06:16 [xTeVe] DVR IP: 172.17.0.3:34400 2021/11/30 05:06:16 [xTeVe] Web Interface: http://172.17.0.3:34400/web/ | xTeVe is also available via the other 2 IP's. 2021/11/30 05:06:16 [xTeVe] XEPG: Create XMLTV file (/config/data/xteve.xml) 2021/11/30 05:06:17 [xTeVe] XEPG: Compress XMLTV file (/config/data/xteve.xml.gz) 2021/11/30 05:06:17 [xTeVe] XEPG: Create M3U file (/config/data/xteve.m3u) 2021/11/30 05:06:17 [xTeVe] XEPG: Ready to use — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

All that is correct. Seems I am connected through the VPN. The Webui is no longer accessible. :(
"The server at 192.168.2.5 is taking too long to respond."

Appreciate your support. :)

**Comment by brycelarge** [comment]
Hello,

Not sure on what your local network is setup but try setting LOCAL_NETWORK=192.168.2.0/24


Kind Regards 
Bryce

**Comment by EvilMonkey0** [comment]
Hello,

I tried that and no luck. As it  does work (thank you for this by the way :) ), I will remove the config file when I need to make updates so I can access the webui. Wish I did not have too, but I have not found another solution.

If you have other ideas, please let me know. Thanks again for your support.

**Comment by EvilMonkey0** [comment]
Hello again Bryce.

I am having another issue. No matter how many streams I set in Xteve, it only allows 1 connection. Any ideas?

**Comment by brycelarge** [comment]
Hello,

In plex or in xteve itself?

I know if you change the streams in xteve you have to remove from plex and re-add.


Kind Regards 
Bryce

**Comment by EvilMonkey0** [comment]
Hello,

It happens in Plex and Jellyfin. Both have 3 streams set and Xteve matches with 3 streams set.

**Comment by EvilMonkey0** [comment]
I have deleted and re-added the DVR. Same results.

**Comment by EvilMonkey0** [comment]
I found the issue and corrected it. Xteve was stuck on single stream for the playlist.

**Comment by EvilMonkey0** [comment]
Correction, the issue still remains. Only 1 stream allowed

**Comment by brycelarge** [comment]
Hello,

 

Unfortunately I do not have any of these issues with xTeVe, I would recommend you ask the amazing authors over on the xTeVe repository, https://github.com/xteve-project/xTeVe

 

If you still having issues browsing to the GUI when OpenVPN is running then perhaps you can give me your docker run script excluding your OpenPVN username and password so I can see if I see anything that stands out. Also your routers internal address, usually 192.168.0.1 or something similar. 

 

Kind Regards

Bryce

 

 

From: EvilMonkey0 ***@***.***>
Reply to: brycelarge/xteve-vpn ***@***.***>
Date: Wednesday, 01 December 2021 at 17:49
To: brycelarge/xteve-vpn ***@***.***>
Cc: ***@***.***" ***@***.***>, Comment ***@***.***>
Subject: Re: [brycelarge/xteve-vpn] Fresh Install (Issue #5)

 

Correction, the issue still remains. Only 1 stream allowed

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
